benchmark/benchmark_beam.py

```diff
@@ -29,7 +28,7 @@ def benchmark_beam_slow_np1(self):
                                      num_elements=num_elements, num_cp=num_cp,
                                      num_load_cases=num_load_cases))
 
-        prob.setup(vector_class=PETScVector)
```

Comment:
So, we don't need to assign the PETScVector anymore when we run in parallel? Wonder if there is a feature doc thata needs updated with that info.

Reply:
Yeah it's no longer necessary. There was a feature doc that showed how to specify use of PETScVectors for a simple model.  I got rid of that section since the user no longer needs to know about it.